src/sentry/models/integration.py

```diff
@@ -48,6 +48,21 @@ class Meta:
         unique_together = (("organization_integration_id", "external_id"),)
 
 
+class RepositoryProjectPathConfig(DefaultFieldsModel):
+    __core__ = False
+
+    repository = FlexibleForeignKey("sentry.Repository")
```

Comment:
@MeredithAnya should we have `db_constraint=False` here as well?

Reply:
It's probably fine, I don't think repository is very high volume. Can't check at the moment :(